platform/windows_x64/docker/install_jre.ps1

```diff
@@ -0,0 +1,23 @@
+$ErrorActionPreference = "Stop"
+$InformationPreference = "Continue"
+if ($env:SCRIPT_DEBUG -ne "") {
+  $VerbosePreference = "Continue"
+}
+# Don't show download progress because invoke-webrequest is slowed down by that. https://stackoverflow.com/a/43477248
+$global:ProgressPreference = "SilentlyContinue"
+Set-StrictMode -Version latest
+
+# Install JRE. Needed to run the EngFlow worker binary.
+Write-Verbose "Installing JRE..."
+Invoke-WebRequest `
+  -Uri 'https://storage.googleapis.com/engflow-tools-public/cdn.azul.com/zulu/bin/zulu17.44.53-ca-jre17.0.8.1-win_x64.zip' `
```

Comment:
nit: store the version in a variable, so we can update it more easily

Reply:
Mind if I skip refactoring here? The iteration time to verify this is pretty long, and I have very little time left before BazelCon.